src/components/formfields/musitModal/__tests__/MusitModal-test.js

```diff
@@ -0,0 +1,31 @@
+import { assert, React, ReactTestUtils } from '../../../../../test/setup';
+import MusitModal from '../MusitModal';
+
+describe('MusitModal', () => {
+  let inputComponent;
+  before('should render Musit Modal', () => {
+    const myDiv = ReactTestUtils.renderIntoDocument(
+      <MusitModal
+        show={Boolean(true)}
+        onHide={(key) => key}
+        onMove={(key) => key}
+        headerText="Hi"
+        loadChildren={(key) => key}
+        loadPath={() => []}
+        clearPath={() => []}
+        loadRoot={(key) => key}
+        setCurrentId={(key) => key}
+        clearCurrentId={(key) => key}
+        rootNode={{}}
+        translate={(key) => key}
+        children={[{ id: 1, name: 'Museum', type: 'Building' }]}
+        path={[{ id: 1, name: 'Museum', type: 'Building' }]}
+      />
+    );
+    inputComponent = ReactTestUtils.scryRenderedDOMComponentsWithTag(myDiv, 'div');
+  });
+
+  it('Check the div', () => {
+    assert(inputComponent[0].innerHTML === '<!-- react-empty: 2 -->')
```

Comment:
I mean, really? you have to read up on the definition of testing. Testing means checking if certain aspects of the code works properly. This test only verifies that you have an empty div somewhere. Thats like, not adequate. Check for the children. Add id's to rows, anything to get at the things you need to test. Even better, add classes to stuff .. and check if there is a method ReactTestUtils.scryRenderedDOMComponentsWithClass or sumtin